packages/client-react-hooks/src/use-nil-compute.ts

```diff
@@ -10,15 +10,31 @@ import {
 import { nilHookBaseResult, UseNilHook } from "./nil-hook-base";
 import { useNillion } from "./use-nillion";
 
-interface ExecuteArgs {
+/** ExecuteArgs is an interface that can be passed to the `execute` function.
+ * @param bindings: `ProgramBindings`
+ * @param values?: `NadaValues`
+ * @param storeIds?: array of `StoreId`s or strings
+ */
+export interface ExecuteArgs {
```

Comment:
Exporting this without adjustment creates type collisions. Shall we rename these after their hooks, eg, `UseNilComputeExecuteArgs`?